modules/core/src/main/scala/org/scalasteward/core/repoconfig/PostUpdateHookConfig.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018-2021 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.repoconfig
+
+import io.circe.Codec
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.deriveConfiguredCodec
+import org.scalasteward.core.data.{ArtifactId, GroupId}
+import org.scalasteward.core.edit.hooks.PostUpdateHook
+import org.scalasteward.core.git.CommitMsg
+import org.scalasteward.core.util.Nel
+
+final case class PostUpdateHookConfig(
+    groupId: Option[GroupId],
+    artifactId: Option[String],
+    command: String,
+    useSandbox: Boolean,
+    commitMessage: String
+) {
+  def toHook: PostUpdateHook =
+    Nel
+      .fromList(command.split(' ').toList)
+      .fold(
+        throw new Exception("Post update hooks must have a command defined.")
```

Comment:
@987Nabil @mzuehlke The exception here would crash Scala Steward hard because it is not caught anywhere. Also `command.split(' ')` cannot handle arguments with spaces, for example if `command` is `sbt "argument1 with spaces" argument2` it would be split into `sbt`, `"argument1`, `with`, `spaces"`, `argument2`. I'm wondering if it would have been easier if `command` would be of type `Nel[String]` so that malformed commands would have been handled by the config file parser and we could support arguments with spaces.

Reply:
@fthomas I was actually think about using `Nel[String]` but was not sure how it would be handled. And I had already an issue with white spaces...
I'll open a follow up pr soon™